.travis.yml

```diff
@@ -2,9 +2,8 @@ sudo: false
 language: scala
 jdk:
   - oraclejdk8
-scala: 2.12.4
```

Comment:
Would rather:

```
scala:
- 2.12.6
- 2.11.12
```

and keep validate logic

Reply:
I'm not sure what's the point of duplicating the Scala version here if it's already defined in the SBT build? `validate.sh` calls `sbt +test` so it will build/test with all versions defined in the `crossScalaVersions` key.

Reply:
Wait... With you suggestion, it's a Travis build matrix, so the `2.11` and `2.12` builds can execute in parallel, right ?

Reply:
Yes